lakefile.lean

```diff
@@ -7,7 +7,7 @@ open Lake DSL
 ## Mathlib dependencies on upstream projects
 -/
 
-require "leanprover-community" / "batteries" @ git "main"
+require "leanprover-community" / "batteries" from git "https://github.com/dtumad/batteries" @ "main"
```

Comment:
I think you can revert this and the change to the `url` in the manifest, leaving only the commit; github does funky stuff and your commit should be available from the upstream repo!

Reply:
I think it's looking for the `rev` on main batteries instead of my fork now though when I changed the manifest URL

Reply:
I guess it never fetches the other commits, even though it's there. I think fine to leave as is and wait for the batteries PR to be merged, then update the SHA to whatever lands.